src/Modules/StatCan.OrchardCore.SaaSConfiguration/Drivers/SaaSConfigurationDisplayDriver.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using OrchardCore.DisplayManagement.Entities;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.OpenId.Services;
+using OrchardCore.Settings;
+using StatCan.OrchardCore.SaaSConfiguration.ViewModels;
+
+namespace StatCan.OrchardCore.SaaSConfiguration
+{
+    public class SaaSConfigurationSettingsDisplayDriver : SectionDisplayDriver<ISite, SaaSConfigurationSettings>
```

Comment:
Maybe move this to a Controller to get more flexibility. Still store these as Site settings but modify them with a controller.